### ISSUE TITLE
any-db@2.2.0 includes an untracked index.d.ts file that fails to build

### ISSUE THREAD

**Issue by mnahkies** [grncdr/node-any-db#76]
It appears that an index.d.ts file that is not tracked in this repository has been published as part of `any-db@2.2.0`. This file fails to build with the following errors:

```
node_modules/any-db/index.d.ts(43,85): error TS2314: Generic type 'Query<Params, Row>' requires 2 type argument(s).
node_modules/any-db/index.d.ts(53,23): error TS2314: Generic type 'Query<Params, Row>' requires 2 type argument(s).
node_modules/any-db/index.d.ts(53,36): error TS2314: Generic type 'Query<Params, Row>' requires 2 type argument(s).
node_modules/any-db/index.d.ts(146,91): error TS2314: Generic type 'Query<Params, Row>' requires 2 type argument(s).
node_modules/any-db/index.d.ts(151,76): error TS2314: Generic type 'Connection<Params>' requires 1 type argument(s).
```

Suggest that either:
* The errors are fixed, typings verified as correct, and committed to repository
* A patch version is published without this index.d.ts file

**Comment by Justus-Maier** [comment]
I got the same issue with any-db@2.2.0; in addition to the mentioned errors I also get this one:
```
TS2305: Module '"C:/project/src/node_modules/any-db/index"' has no exported member 'createConnection'.
```
Installing any-db@2.1 solves the issue.

**Comment by grncdr** [comment]
Fixed in v2.2.1 and v2.2.0 yanked
